core/events/events_comment_move.ts

```diff
@@ -67,15 +65,12 @@ export class CommentMove extends CommentBase {
    * called once.
```

Comment:
I can't leave a comment there but on line 57 the comment for `this.comment_` needs to be updated.